src/herder/test/PendingEnvelopesTests.cpp

```diff
@@ -28,6 +28,7 @@ TEST_CASE("PendingEnvelopes recvSCPEnvelope", "[herder]")
     auto& pk = s.getPublicKey();
     cfg.QUORUM_SET.validators.emplace_back(s.getPublicKey());
     Application::pointer app = createTestApplication(clock, cfg);
+    app->start();
```

Comment:
Why do we need to start the application?
Should this be done by `createTestApplication` - it sounds like a bug to use the app without starting it (I guess there might be a few places where we want to do this, but maybe those places should be the ones passing a `start=false` parameter to `createTestApplication`?

Reply:
good idea, updated